<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Lato</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>DGPS Message Generator</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="Main_box">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="refPos_label">
          <property name="text">
           <string>Reference Position</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="refPosSet_button">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refPosMeasure_button">
            <property name="text">
             <string>Measure</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="refPosSave_button">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refPosLoad_button">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="Status_output"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <widget class="QLineEdit" name="Baud_input">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Baud_label">
          <property name="text">
           <string>Baud:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Port_label">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Serial Port:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="Output_label">
          <property name="text">
           <string>Output</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="Input_label">
          <property name="text">
           <string>Input</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="Baud_output"/>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="Port_output"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="Port_input">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="Start_button">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Stop_button">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>Port_input</tabstop>
  <tabstop>Port_output</tabstop>
  <tabstop>Baud_input</tabstop>
  <tabstop>Baud_output</tabstop>
  <tabstop>Start_button</tabstop>
  <tabstop>Stop_button</tabstop>
  <tabstop>refPosSet_button</tabstop>
  <tabstop>refPosMeasure_button</tabstop>
  <tabstop>refPosSave_button</tabstop>
  <tabstop>refPosLoad_button</tabstop>
  <tabstop>Status_output</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Port_input</sender>
   <signal>returnPressed()</signal>
   <receiver>Start_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Baud_input</sender>
   <signal>returnPressed()</signal>
   <receiver>Start_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Port_output</sender>
   <signal>returnPressed()</signal>
   <receiver>Start_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Baud_output</sender>
   <signal>returnPressed()</signal>
   <receiver>Start_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
